tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7651,7 +7651,7 @@ def err_hlsl_unsupported_keyword_for_version : Error<
 def err_hlsl_unsupported_keyword_for_min_precision : Error<
    "%0 is only supported with -enable-16bit-types option">;
 def err_hlsl_unsupported_template_for_intrinsic: Error<
-   "Unrecognized template arguments for intrinsic %0">;
+   "Unrecognized template arguments for intrinsic %0, %1 supported.">;
```

Comment:
I know I recommended this, but I think multiple messages are better.  Since I'm asking for other changes, I suppose this one should be done while you have a chance.  Here are three examples:
```
def err_hlsl_intrinsic_template_arg_unsupported: Error<
   "Explicit template arguments on intrinsic %0 are not supported.">;
`def err_hlsl_intrinsic_template_arg_requires_2018: Error<
   "Explicit template arguments on intrinsic %0 requires HLSL version 2018 or above.">;
def err_hlsl_intrinsic_template_arg_scalar_vector_16: Error<
   "Explicit template arguments on intrinsic %0 are limited one to scalar or vector type up to 16 bytes in size.">;
``